inginious/frontend/templates/course_admin/task_dispensers/util_task_edit_modal.html

```diff
@@ -54,6 +54,14 @@ <h5 class="modal-title">{{_("Edit Task")}}</h5>
                     </div>
                 </div>
             </div>
+
+            <div style="margin-top: 15px;">{{_("Categories")}}</div>
```

Comment:
You can use the ``mt-X`` CSS class where X is 1,2,3... to apply top margins the boostrap way.